### PR TITLE
Update wrong placeholders in ar-arb translation

### PR DESCRIPTION
**Pull request by quintanilhar**
## Ticket
https://oat-sa.atlassian.net/browse/TR-3026

## Summary
Replacement characters (%s, %d, …) are not replaced in translations strings rendered.

## Dependencies
No dependencies

## Sandbox environment
TODO

### CODE REVIEW

File: locales/ar-arb/messages.po [hunk 1]
```diff
@@ -251,7 +254,7 @@ msgid "The identifier of the choice. This identifier must not be used by any oth
 msgstr "معرّف الاختيار. يجب عدم استخدام هذا المعرف من قبل أي استجابة أخرى أو متغير بند. المعرّف هو سلسلة من الأحرف التي يجب أن تبدأ بحرف أو شرطة سفلية (\"_\") وتحتوي فقط على أحرف ، وشرطات سفلية ، وواصلات (\"-\") ، ونقطة (\".\" ، ويعرف أيضًا باسم النقطة الكاملة) ، وأرقام ، والجمع بين الأحرف والموسعات."
 
 msgid "There is already one interaction with the same identifier \"%s\" (label : \"%s\") and same version : %s. Do you want to override the existing one ?"
-msgstr "يوجد بالفعل تفاعل واحد مع نفس المعرف \"٪ s\" (التسمية: \"٪ s\") ونفس الإصدار:٪ s. هل تريد تجاوز الموجود؟"
+msgstr "يوجد بالفعل تفاعل واحد مع نفس المعرف  \"%s\" (التسمية: \"%s\") ونفس الصيغة: %s . هل تريد تجاوز الموجود?"
```
Comment (wazelin) [inline locales/ar-arb/messages.po:257]: Are you absolutely sure about that question mark?
I'm no expert in Arabic, but Google puts the question mark on the left (at the end of the sentence).
https://translate.google.com/?sl=auto&tl=ar&text=Do%20you%20want%20to%20override%20the%20existing%20one%3F&op=translate

Let's re-check it with a help of a PM.

Same about the full stop mark in the first sentence (I know it's like that in the original copy, though).

**Review by wazelin — CHANGES_REQUESTED** [review]

File: locales/ar-arb/messages.po (hunk 1 above)
Comment (quintanilhar) [reply locales/ar-arb/messages.po:257]: Not at all. These translations were provided by the customer. What you said makes sense, perhaps we can revert back to the Arabian question mark symbol. This is how it looks at the moment:

image

**Review by marpesia — APPROVED** [review]
LGTM

File: locales/ar-arb/messages.po (hunk 1 above)
Comment (quintanilhar) [reply locales/ar-arb/messages.po:257]: Fixed: 
image

**Review by wazelin — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Pull request merged**